src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -182,18 +182,33 @@ object ArtifactOrProject {
   }
 }
 
-case class Sha1Value(toHex: String)
+sealed trait DigestType {
+  def getDigestInstance: MessageDigest
+  def expectedLength: Int
```

Comment:
can we say `expectedHexLength` or something to be clear this is not bits or bytes?